src/app/components/Billboard/index.tsx

```diff
@@ -0,0 +1,110 @@
+/** @jsx jsx */
+import { useContext, forwardRef } from 'react';
+import { jsx } from '@emotion/react';
+import useViewTracker from '#app/hooks/useViewTracker';
+import useClickTrackerHandler from '#app/hooks/useClickTrackerHandler';
+import { EventTrackingMetadata } from '#app/models/types/eventTracking';
+import idSanitiser from '#app/lib/utilities/idSanitiser';
+import Heading from '../Heading';
+import LiveLabel from '../LiveLabel';
+import MaskedImage from '../MaskedImage';
+import { ServiceContext } from '../../contexts/ServiceContext';
+import styles from './index.styles';
+import Text from '../Text';
+
+interface BillboardProps {
+  heading: string;
+  description?: string;
+  lang: string;
+  link?: string;
+  image: string;
+  eventTrackingData?: EventTrackingMetadata;
+  showLiveLabel: boolean;
+  imageUrl?: string;
+  imageUrlTemplate?: string;
+  imageWidth?: number;
+}
+
+const Banner = forwardRef(
+  (
+    { heading, description, link, image, imageUrl, imageUrlTemplate, imageWidth, eventTrackingData, showLiveLabel, lang }: BillboardProps,
+    viewRef,
+  ) => {
+    const { dir } = useContext(ServiceContext);
+    const isRtl = dir === 'rtl';
+
+    // temporarily invoked to resolve GH errors - remove this when we start using
+    // the service context properly 
+    isRtl;
```

Comment:
Looks like we don't need this (I tested it on Storybook, and the RTL behaviour just works), so we can delete.